packages/client/lib/util/debug.ts

```diff
@@ -0,0 +1,65 @@
+import { Block } from '@ethereumjs/block'
+import { VMExecution } from '../sync/execution'
+
+/**
+ * Generates a code snippet which can be used to replay an erraneous block
+ * locally in the VM
+ *
+ * @param block
+ */
+export async function debugCodeReplayBlock(execution: VMExecution, block: Block) {
+  const code = `
+/**
+ * Script for locally executing a block in the EthereumJS VM,
+ * meant to be used from packages/vm directory within the
+ * https://github.com/ethereumjs/ethereumjs-monorepo repository.
+ * 
+ * Block: ${block.header.number}
+ * Hardfork: ${execution.hardfork}
+ * 
+ * Run with: DEBUG=vm:*:*,vm:*,-vm:ops:* ts-node [SCRIPT_NAME].ts
```

Comment:
Should be `export DEBUG`. If I don't use export then debug is not triggered.

Reply:
Ah, no, export should not be needed. Your are likely putting an `&&` in between the commands?